python/ray/train/jax/utils.py

```diff
@@ -0,0 +1,25 @@
+from ray.air.config import ScalingConfig
+
+
+def ensure_tpu_resources_capitalized(scaling_config: ScalingConfig) -> ScalingConfig:
+    """Ensure that the resources_per_worker has capitalized keys."""
+    # case-insensitivize
+    # since `tpu` is not the standard resources in ray currently
+    # add these lines to prevent the cases where the users
+    # give the lower-case `tpu` as the resources
+    # and change the key to upper case!
```

Comment:
Actually, thinking about this some more do we even need this function? Instead, we should just document that TPU scheduling is case-sensitive (similar to what is done for CPU and GPU in `ScalingConfig.resources_per_worker`).